app/controllers/api/v1/items/search_controller.rb

```diff
@@ -1,11 +1,24 @@
 class Api::V1::Items::SearchController < ApplicationController
 
+  def index
```

Comment:
I would really like to remove the repetition in the search controllers and search query services but right now I just need to get everything working!